src/xFactor/config/AppConfig.java

```diff
@@ -0,0 +1,60 @@
+package xFactor.config;
+
+import java.util.Properties;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+
+import xFactor.infrastructure.model.Judge;
+import xFactor.infrastructure.model.JudgeFavourite;
+import xFactor.infrastructure.model.Participant;
+import xFactor.infrastructure.model.Stage;
+import xFactor.infrastructure.model.Vote;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages = "xFactor")
+public class AppConfig extends WebMvcConfigurerAdapter {
+	private SessionFactory sessionFactory = new org.hibernate.cfg.Configuration()

```

Comment:
Didn't mean this. You need to create session bean here, so then you say @Autowire session, then it will check from the context and get it from there